README.md

```diff
@@ -1 +1,65 @@
-# framework-awesomebook
\ No newline at end of file
+# framework-awesomebook
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# awesomebook
+
+> This a group work project
+
+# WORKFLOW SCREENSHOT
+![screenshot]()
+
+## Built With
+- HTML
+- CSS
+- Figma Tempalate
+## Live Demo (if available)
+
+[Live Demo Link](https://siansandatamara.github.io/framework-awesomebook/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Web Browser
+- Code Editor
+- Figma Template
+- Prior knowledge of CSS, HTML, GIT, GITHUB and Linters.
+
+### Setup
+
+```bash
+git clone// https://siansandatamara.github.io/awesome-book/
+cd awsomebook
+
+### Usage
+
+Open the index file of the project in the browser to see the project
+
+## Authors
+   ** cyril muchemi**
+👤 **Tamara Siansanda**
+
+- GitHub: [@SiansandaTamara](https://github.com/SiansandaTamara)
+- Twitter: [@twitterhandle](https://twitter.com/TamaraSiansanda)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] You wrapped all sections as a code. Please update this part and your readme file look professional. 

![image](https://user-images.githubusercontent.com/38635158/164336014-7f949891-0de8-44dd-ba05-68c28884b38f.png)